components/yaml.rst

```diff
@@ -250,6 +250,59 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Syntax Validation
+~~~~~~~~~~~~~~~~~
+
+The syntax of YAML contents can be validated through the CLI using the
+:class:`Symfony\\Component\\Yaml\Command\\LintCommand` command.
```

Comment:
Not sure what is the best way to do it, nor where we can encounter similar cases in the components documentation... Simply adding a little sentence followed by:

```sh
composer require symfony/console
```

?

I quickly searched and found those documentations:

- https://github.com/symfony/symfony-docs/blob/master/console/command_in_controller.rst#showing-colorized-command-output
- https://github.com/symfony/symfony-docs/blob/master/setup/bundles.rst#look-for-deprecations-and-fix-them
- https://github.com/symfony/symfony-docs/blob/master/service_container/lazy_services.rst#installation
- https://github.com/symfony/symfony-docs/blob/master/components/form.rst#csrf-protection